New version: Microsoft.OneDrive.Internal.Slow version 22.225.1026.0001

vedantmgoyal9 opened the pull request:
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Microsoft OneDrive Internal Slow |     |
| Version     | 22.225.1026.0001     |  |
| Publisher   | Microsoft Corporation   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [3487](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3342713316>)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86638)

vedantmgoyal9 committed:
New version: KristenMcWilliam.Nyrna version 2.6.1 (#87417)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: wire.wire version 3.29.4349 (#87375)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: JFrog.Conan version 1.54.0 (#87376)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: OpenRPAApS.OpenRPA version 1.4.47.0 [FP-O] (#87514)

* New version: OpenRPAApS.OpenRPA version 1.4.46.0

* Revert "New version: OpenRPAApS.OpenRPA version 1.4.46.0"

This reverts commit 92203eef03e9d1131b2f2fe2f0ffbb2435064021.

* New version: OpenRPAApS.OpenRPA version 1.4.47.0

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

solomoncyj committed:
New package: UrBackup.UrBackup.Client version 2.5.20 (#87526)

* New package: UrBackup.UrBackup.Client version 2.5.20

* Update UrBackup.UrBackup.Client.locale.en-US.yaml

* Update UrBackup.UrBackup.Client.locale.en-US.yaml

* Apply suggestions from code review

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

vedantmgoyal9 committed:
New version: Wondershare.DemoCreator version 6.2.1 (#87419)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: JetBrains.PyCharm.Community.EAP version 223.7401.13 [FP-D] (#87425)

* New version: JetBrains.PyCharm.Community.EAP version 223.7401.13

* Update manifests/j/JetBrains/PyCharm/Community/EAP/223.7401.13/JetBrains.PyCharm.Community.EAP.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

vedantmgoyal9 committed:
New version: JetBrains.GoLand.EarlyAccess version 223.7401.12 [FP-D] (#87426)

* New version: JetBrains.GoLand.EarlyAccess version 223.7401.12

* Fix ProductCode

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

vedantmgoyal9 committed:
New version: Discord.Discord.Development version 1.0.682 (#87427)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Discord.Discord.Development version 1.0.681 [FP-O] (#87432)

* New version: Discord.Discord.Development version 1.0.680

* Revert "New version: Discord.Discord.Development version 1.0.680"

This reverts commit e1458d8de433e5c04439f8ea3ae011c00e972d92.

* New version: Discord.Discord.Development version 1.0.681

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: JetBrains.IntelliJIDEA.Community.EAP version 223.7401.7 (#87450)

* New version: JetBrains.IntelliJIDEA.Community.EAP version 223.7401.7

* Update manifests/j/JetBrains/IntelliJIDEA/Community/EAP/223.7401.7/JetBrains.IntelliJIDEA.Community.EAP.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

vedantmgoyal9 committed:
New version: JetBrains.WebStorm.EAP version 223.7401.14 (#87452)

* New version: JetBrains.WebStorm.EAP version 223.7401.14

* Update manifests/j/JetBrains/WebStorm/EAP/223.7401.14/JetBrains.WebStorm.EAP.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

* Update manifests/j/JetBrains/WebStorm/EAP/223.7401.14/JetBrains.WebStorm.EAP.locale.en-US.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

* Update manifests/j/JetBrains/WebStorm/EAP/223.7401.14/JetBrains.WebStorm.EAP.locale.en-US.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

* Update manifests/j/JetBrains/WebStorm/EAP/223.7401.14/JetBrains.WebStorm.EAP.locale.en-US.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

wingetbot committed:
Automatic update of Microsoft.PowerBI from 2.110.805.0 to 2.110.1085.0 (#87481)

* Creating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.installer.yaml

* Creating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.locale.en-US.yaml

* Creating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.yaml

* Updating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.installer.yaml

* Updating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.locale.en-US.yaml

* Updating manifests/m/Microsoft/PowerBI/2.110.1085.0/Microsoft.PowerBI.yaml

vedantmgoyal9 committed:
New version: calibre.calibre.portable version 6.8.0.0 (#87488)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Oxen.Session version 1.10.4 (#87390)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: KDE.Dolphin version 22.08.3 [FP-O] (#87494)

* New version: KDE.Dolphin version 22.08.3

* Revert "New version: KDE.Dolphin version 22.08.3"

This reverts commit 7671bd2405d3f0e5d2d257ea370922f75dd50ad1.

* New version: KDE.Dolphin version 22.08.3

* Revert "New version: KDE.Dolphin version 22.08.3"

This reverts commit 34a206d2ab52b69d4491a00ceb0c6f8f966613c8.

* New version: KDE.Dolphin version 22.08.3

* Revert "New version: KDE.Dolphin version 22.08.3"

This reverts commit a8d33b11ded8ba9f6fdd8e31cb279cef79bc538d.

* New version: KDE.Dolphin version 22.08.3

* Revert "New version: KDE.Dolphin version 22.08.3"

This reverts commit 9988c766f1fd4f3546375ad74928d0c6a5930b75.

* New version: KDE.Dolphin version 22.08.3

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: KDE.Kate version 22.08.3 [FP-O] (#87495)

* New version: KDE.Kate version 22.08.3

* Revert "New version: KDE.Kate version 22.08.3"

This reverts commit 7dca2f619e5a0530e66cd5dec0241029bb84e7bd.

* New version: KDE.Kate version 22.08.3

* Revert "New version: KDE.Kate version 22.08.3"

This reverts commit 5e4a4c189066c57cfc4c599f7f7aef62e0e5a273.

* New version: KDE.Kate version 22.08.3

* Revert "New version: KDE.Kate version 22.08.3"

This reverts commit 93be30efa5ad67fd045ea0d0d71ac08d45a8a403.

* New version: KDE.Kate version 22.08.3

* Revert "New version: KDE.Kate version 22.08.3"

This reverts commit 3a38527d96db5aad5c2f6854404a22789c0feda1.

* New version: KDE.Kate version 22.08.3

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: KDE.Kdenlive version 22.08.3 [FP-O] (#87497)

* New version: KDE.Kdenlive version 22.08.3

* Revert "New version: KDE.Kdenlive version 22.08.3"

This reverts commit 50e7cadae618051d6c55da80c277764d1fdacded.

* New version: KDE.Kdenlive version 22.08.3

* Revert "New version: KDE.Kdenlive version 22.08.3"

This reverts commit d646e4889d23fa68da6c6b2f5a34a709e7ac7d5f.

* New version: KDE.Kdenlive version 22.08.3

* Revert "New version: KDE.Kdenlive version 22.08.3"

This reverts commit 1cc6966dd9303b3f787106fe6097dec0a28ab2ee.

* New version: KDE.Kdenlive version 22.08.3

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: KDE.Okular version 22.08.3 [FP-O] (#87498)

* New version: KDE.Okular version 22.08.3

* Revert "New version: KDE.Okular version 22.08.3"

This reverts commit 90a99febdd39bbd17a5475c6edd0b3fe1f7cce81.

* New version: KDE.Okular version 22.08.3

* Revert "New version: KDE.Okular version 22.08.3"

This reverts commit 66f0f187ffecf499fff58442b2fc4aae4265e3b8.

* New version: KDE.Okular version 22.08.3

* Revert "New version: KDE.Okular version 22.08.3"

This reverts commit 57ab08588b942af52a6b1d0d7dbb925ff42e1554.

* New version: KDE.Okular version 22.08.3

* Revert "New version: KDE.Okular version 22.08.3"

This reverts commit 2b59aa6179f7f8db7886cdf2216c88fbcebc3d95.

* New version: KDE.Okular version 22.08.3

* Revert "New version: KDE.Okular version 22.08.3"

This reverts commit d80d22f055343a701877863c4b7fe6cecd3bbd05.

* New version: KDE.Okular version 22.08.3

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: KDE.KDEConnect version 22.08.3 [FP-O] (#87496)

* New version: KDE.KDEConnect version 22.08.3

* Revert "New version: KDE.KDEConnect version 22.08.3"

This reverts commit 7a1e3bc09a9a1b5af86ecc61938f3f8a933b30bf.

* New version: KDE.KDEConnect version 22.08.3

* Revert "New version: KDE.KDEConnect version 22.08.3"

This reverts commit 30245b41f591b02f5b90dc53836b00fd551bc605.

* New version: KDE.KDEConnect version 22.08.3

* Revert "New version: KDE.KDEConnect version 22.08.3"

This reverts commit 3679b336e3854652a88005559813e0a93fb4044f.

* New version: KDE.KDEConnect version 22.08.3

* Revert "New version: KDE.KDEConnect version 22.08.3"

This reverts commit 0aeb5e5854cb0ccc8af506dfc9ba737ef10f2ebb.

* New version: KDE.KDEConnect version 22.08.3

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

solomoncyj committed:
New package: UrBackup.UrBackup.Server version 2.5.26 (#87527)

* New package: UrBackup.UrBackup.Server version 2.5.26

* Apply suggestions from code review

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

vedantmgoyal9 committed:
New version: RandomEngy.VidCoder.Beta version 8.10 (#86640)

* New version: RandomEngy.VidCoder.Beta version 8.10

* Update RandomEngy.VidCoder.Beta.locale.en-US.yaml

* Update RandomEngy.VidCoder.Beta.installer.yaml

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

Trenly committed:
Add Schema Header (#83863)

Trenly committed:
New version: Microsoft.Sysinternals.ProcessExplorer version 17.01 (#87464)

vedantmgoyal9 committed:
New version: mentebinaria.retoolkit version 2022.10 (#86620)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

anamnavi committed:
New version: Microsoft.PowerShell version 7.3.101.0 (7.3.0-rc1) (#87531)

* New version: Microsoft.PowerShell version 7.3.101.0

* move 7.3.101.0 to Preview folder, and remove ProductCode for msix packages

* also comment out ReleaseNotes which was accidentally added

* fix PackageIdentifier

* package identifier was missing in file name

* add optional MinimumOSVersion which is required

vedantmgoyal9 committed:
New version: Google.Chrome.Canary version 109.0.5407.0 [FP-O] (#87491)

* New version: Google.Chrome.Canary version 109.0.5399.3

* Revert "New version: Google.Chrome.Canary version 109.0.5399.3"

This reverts commit 0e956e1f4cecd7fba158f1523cfc83b3ea3c596f.

* New version: Google.Chrome.Canary version 109.0.5401.0

* Revert "New version: Google.Chrome.Canary version 109.0.5401.0"

This reverts commit f81f04c4b8de0b83de95c61cc82ff13062e51649.

* New version: Google.Chrome.Canary version 109.0.5402.0

* Revert "New version: Google.Chrome.Canary version 109.0.5402.0"

This reverts commit f1a205f18a8b308999f75cea5af34beb81abc4e7.

* New version: Google.Chrome.Canary version 109.0.5403.0

* Revert "New version: Google.Chrome.Canary version 109.0.5403.0"

This reverts commit ac436dee9b216dc33c24d8fac1ec1dc7a07abb65.

* New version: Google.Chrome.Canary version 109.0.5406.0

* Revert "New version: Google.Chrome.Canary version 109.0.5406.0"

This reverts commit 603e102ee2715bf75add133f8cb16949ad5f285a.

* New version: Google.Chrome.Canary version 109.0.5407.0

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: JetBrains.Gateway version 2022.2.4 (#87374)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Automatic deletion of LINE.LINE 7.13.1.2875 (#87581)

* Updating manifests/l/LINE/LINE/7.13.1.2875/LINE.LINE.installer.yaml

* Updating manifests/l/LINE/LINE/7.13.1.2875/LINE.LINE.locale.en-US.yaml

* Updating manifests/l/LINE/LINE/7.13.1.2875/LINE.LINE.yaml

vedantmgoyal9 committed:
New version: JetBrains.PyCharm.Professional.EAP version 223.7401.13 [FP-D] (#87424)

* New version: JetBrains.PyCharm.Professional.EAP version 223.7401.13

* Apply suggestions from code review

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

vedantmgoyal9 committed:
New version: JetBrains.IntelliJIDEA.Ultimate.EAP version 223.7401.7 (#87451)

* New version: JetBrains.IntelliJIDEA.Ultimate.EAP version 223.7401.7

* Update manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/223.7401.7/JetBrains.IntelliJIDEA.Ultimate.EAP.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>
Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

vedantmgoyal9 committed:
New version: alagrede.znote version 1.7.9 (#87377)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
Update: KDE.KDEConnect version 22.08.3 (#87373)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: vcync.modV version 3.25.3 (#87407)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: WeMod.WeMod.Beta version 8.3.9-beta00 (#87413)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

Trenly committed:
Normalize Kiwix.WikiMed 2.0.4.0 (#75891)

* Normalize Kiwix.WikiMed 2.0.4.0

* Try appx

Jaifroid committed:
Kiwix.WikiMed version 2.1.8.0 (#82540)

ilike2burnthing committed:
New version: Jackett.Jackett version 0.20.2220 (#87746)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Commit:
New version: JAMSoftware.TreeSize version 8.5.1 (#87755)

Commit:
New version: Facebook.Messenger version 168.0.415453512 (#87751)

lordyavin committed:
ShiningLight.OpenSSL version 3.0.7 (#87756)

wingetbot committed:
Automatic update of Quicken.Quicken from 27.1.41.10 to 27.1.44.28 (#87753)

* Creating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.installer.yaml

* Creating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.locale.en-US.yaml

* Creating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.yaml

* Updating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.installer.yaml

* Updating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.locale.en-US.yaml

* Updating manifests/q/Quicken/Quicken/27.1.44.28/Quicken.Quicken.yaml

SpecterShell committed:
New version: Catsxp.Catsxp version 107.2.11.3 (#87757)

Commit:
New version: VivaldiTechnologies.Vivaldi version 5.5.2805.42 (#87774)

Commit:
Remove version: uvncbvba.UltraVnc version 1.3.8.2 (#87767)

Commit:
New version: FireAlpaca.FireAlpaca version 2.8.11 (#87777)

SpecterShell committed:
Moved: Alibaba.AlipayKeyTool version 1.0.11 (#87780)

SpecterShell committed:
Moved: Alibaba.AlipayKeyTool version 1.0.7 (#87782)

SpecterShell committed:
Moving: Alibaba.AlipayDevelopmentAssistant version 1.0.11 (#87784)

SpecterShell committed:
Moving: Alibaba.AlipayDevelopmentAssistant version 1.0.7 (#87786)

SpecterShell committed:
Moved: Alibaba.AlipayKeyTool version 1.0.12 (#87779)

SpecterShell committed:
New version: Alibaba.AlipayKeyTool version 2.0.2 (#87788)

SpecterShell committed:
New version: RoyQu.RedPanda-C++ version 2.3 (#87771)

wingetbot committed:
Automatic update of ViewSonic.myViewBoard.Whiteboard from 2.60.3.0 to 2.61.4.0 (#87773)

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.installer.yaml

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.locale.en-US.yaml

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.installer.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.locale.en-US.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.4.0/ViewSonic.myViewBoard.Whiteboard.yaml

SpecterShell committed:
New version: Baidu.BaiduPinyin version 5.8.8.16 (#87789)

* New version: Baidu.BaiduPinyin version 5.8.8.16

* Change PublisherSupportUrl

SpecterShell committed:
New version: 360.360SE version 13.1.6320.0 (#87790)

SpecterShell committed:
ReleaseNotes: B3log.SiYuan version 2.4.10 (#87772)

SpecterShell committed:
Moving: Alibaba.AlipayDevelopmentAssistant version 1.0.10 (#87783)

SpecterShell committed:
Moved: Alibaba.AlipayKeyTool version 1.0.10 (#87781)

SpecterShell committed:
Moving: Alibaba.AlipayDevelopmentAssistant version 1.0.12 (#87785)

Commit:
New version: VivaldiTechnologies.Vivaldi.Snapshot version 5.6.2829.3 (#87775)

SpecterShell committed:
ReleaseNotes: GyDi.ClashVerge version 1.1.1 (#87791)

* ReleaseNotes: GyDi.ClashVerge version 1.1.1

* Add AppsAndFeaturesEntries

SpecterShell committed:
ReleaseNotes: GyDi.ClashVerge version 1.1.2 (#87792)

* ReleaseNotes: GyDi.ClashVerge version 1.1.2

* Add AppsAndFeaturesEntries

SpecterShell committed:
New version: Alibaba.AlipayKeyTool version 2.0.1 (#87787)

* New version: Alibaba.AlipayKeyTool version 2.0.1

* Replace asterisk

SpecterShell committed:
New version: Alibaba.AlipayKeyTool version 2.0.0 (#87776)

* New version: Alibaba.AlipayKeyTool version 2.0.0

* Replace asterisks

SpecterShell committed:
Modify: mentebinaria.retoolkit version 2022.10 (#87778)

SpecterShell committed:
ReleaseNotes: Tencent.TranSmart version Alpha0.8.6 (#87761)

SpecterShell committed:
ReleaseNotes: FengHe.FocusNote version 2.16.2-679 (#87760)

Commit:
New version: Notepad++.Notepad++ version 8.4.7 (#87805)

stephengillie commented on manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe
+  InstallerSha256: 6513912FB6DAA99ABD1062055F6D9E0FD626A599336BF45FEFEB2631EEB1F929
+- Architecture: arm64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/OneDriveSetup.exe
```
InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe

stephengillie commented on manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe
+  InstallerSha256: E35C583CA23AF164FF91D36CD299C9FE006FDF7CABEC547ED5CBA5BE262A441C
+- Architecture: x86
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe
```
InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/OneDriveSetup.exe

stephengillie commented on manifests/m/Microsoft/OneDrive/Internal/Slow/22.225.1026.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/arm64/OneDriveSetup.exe
```
https://oneclient.sfx.ms/Win/MsitSlow/22.225.1026.0001/amd64/OneDriveSetup.exe

wingetbot committed:
Updating manifests/s/sbt/sbt/.validation (#87016)

vedantmgoyal9 committed:
New version: ByteDance.StreamingTool version 3.9.3 (#87486)

* New version: ByteDance.StreamingTool version 3.9.3

* Update ByteDance.StreamingTool.locale.en-US.yaml

* Update ByteDance.StreamingTool.locale.zh-CN.yaml

* Update ByteDance.StreamingTool.yaml

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

Commit:
New package: RawTherapee.RawTherapee.Dev version 5.8-3162-g11168aa64 (#87806)

dotnet-winget-bot committed:
Microsoft.DotNet.Runtime.6 version 6.0.11 (#87715)

dotnet-winget-bot committed:
Microsoft.DotNet.HostingBundle.6 version 6.0.11 (#87717)

dotnet-winget-bot committed:
Microsoft.DotNet.SDK.6 version 6.0.306 (#87720)

dotnet-winget-bot committed:
Microsoft.DotNet.SDK.6 version 6.0.111 (#87721)

dotnet-winget-bot committed:
Microsoft.DotNet.AspNetCore.3_1 version 3.1.31 (#87801)

dotnet-winget-bot committed:
Microsoft.DotNet.HostingBundle.3_1 version 3.1.31 (#87802)

dotnet-winget-bot committed:
Microsoft.DotNet.SDK.3_1 version 3.1.425 (#87804)

dotnet-winget-bot committed:
Microsoft.DotNet.DesktopRuntime.3_1 version 3.1.31 (#87803)

dotnet-winget-bot committed:
Microsoft.DotNet.Runtime.3_1 version 3.1.31 (#87800)

SpecterShell committed:
New version: Baidu.BaiduWenku version 1.1.4 (#61481)

* New version: Baidu.BaiduWenku version 1.1.4

* Update manifest

* Add Protocols

Trenly committed:
Add Schema Header (#83819)

nebula-it committed:
Add talosctl (#87049)

* Add talosctl

* Add Argocd

* Revert "Add Argocd"

This reverts commit 844b3c3d2626b7c697b39f5d634768ca71fca1a5.

SpecterShell committed:
New version: Microsoft.PCManager version 1.2.4.6 (#87799)

vedantmgoyal9 committed:
New version: Telerik.Fiddler.Everywhere.Insiders version 4.0.0-insiders (#87360)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Mozilla.Thunderbird.Beta version 107.0b4 (#87813)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: swiftyapp.swifty version 0.6.10 (#87361)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
Update: OpenJS.NodeJS.Nightly version 20.0.0 (#87362)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Google.Chrome.Canary version 109.0.5408.0 (#87364)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: GeekCorner.threema version 2.5.1 (#87365)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: eMClient.eMClient version 9.2.1185.0 (#87357)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Elgato.4KCaptureUtility version 1.7.9.4936 (#87358)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

SpecterShell committed:
New version: RoyQu.RedPanda-C++ version 2.2 (#87770)

crutkas committed:
Microsoft.PowerToys version 0.64.1 (#87811)

SpecterShell committed:
Modify: Versa.MagiCut version 1.2.0 (#64045)

* Update: Versa.MagiCut version 1.2.0

* Try decoded URL

* Update manifest

vedantmgoyal9 committed:
New version: JetBrains.Rider version 2022.2.4 (#87345)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Automatic update of Cisco.WebexTeams from 42.9.0.23494 to 42.10.0.23814 (#87549)

* Creating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.installer.yaml

* Creating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.locale.en-US.yaml

* Creating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.yaml

* Updating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.installer.yaml

* Updating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.locale.en-US.yaml

* Updating manifests/c/Cisco/WebexTeams/42.10.0.23814/Cisco.WebexTeams.yaml

Trenly committed:
Remove: AnyDeskSoftwareGmbH.AnyDeskMSI version 7.1.6 (#87564)

wingetbot committed:
Automatic deletion of Kingsoft.WPSOffice.CN 11.1.0.11369 (#87578)

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11369/Kingsoft.WPSOffice.CN.installer.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11369/Kingsoft.WPSOffice.CN.locale.en-US.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11369/Kingsoft.WPSOffice.CN.locale.zh-CN.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11369/Kingsoft.WPSOffice.CN.yaml

dotnet-winget-bot committed:
Microsoft.DotNet.SDK.6 version 6.0.403 (#87719)

vedantmgoyal9 committed:
Update: KDE.Dolphin version 22.08.3 (#87371)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

Trenly committed:
Update: Rustlang.Rust.MSVC version 1.65.0 (#87472)

* Update: Rustlang.Rust.MSVC version 1.65.0

* Fix key order

Trenly committed:
Update: Rustlang.Rust.GNU version 1.65.0 (#87544)

vedantmgoyal9 committed:
Update: KDE.Kate version 22.08.3 (#87368)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Discord.Discord.PTB version 1.0.1020 (#87372)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Creating manifests/r/RoyQu/RedPanda-C++/.validation (#87825)

Commit:
New version: VMware.WorkstationPro version 16.2.4 (#67201)

vedantmgoyal9 committed:
New version: Kingsoft.KDocs version 3.7.9 (#87363)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

dotnet-winget-bot committed:
Microsoft.DotNet.AspNetCore.6 version 6.0.11 (#87716)

Commit:
New version: VMware.WorkstationPlayer version 16.2.4 (#67200)

vedantmgoyal9 committed:
Update: KDE.Okular version 22.08.3 (#87367)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Creating manifests/r/RawTherapee/RawTherapee/Dev/.validation (#87824)

dotnet-winget-bot committed:
Microsoft.DotNet.DesktopRuntime.6 version 6.0.11 (#87718)

wingetbot committed:
Automatic deletion of Kingsoft.WPSOffice.CN 11.1.0.11294 (#87576)

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11294/Kingsoft.WPSOffice.CN.installer.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11294/Kingsoft.WPSOffice.CN.locale.en-US.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11294/Kingsoft.WPSOffice.CN.locale.zh-CN.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11294/Kingsoft.WPSOffice.CN.yaml

vedantmgoyal9 committed:
Update: KDE.Kdenlive version 22.08.3 (#87369)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

sprout2000 committed:
New version: sprout2000.LeafView version 2.2.11 (#87580)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

stephengillie commented:
@wingetbot waivers Add Validation-Installation-Error

vedantmgoyal9 committed:
New version: Microsoft.Bicep version 0.12.40.16777 (#87370)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Automatic deletion of Kingsoft.WPSOffice.CN 11.1.0.11744 (#87582)

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11744/Kingsoft.WPSOffice.CN.installer.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11744/Kingsoft.WPSOffice.CN.locale.en-US.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11744/Kingsoft.WPSOffice.CN.locale.zh-CN.yaml

* Updating manifests/k/Kingsoft/WPSOffice/CN/11.1.0.11744/Kingsoft.WPSOffice.CN.yaml

SpecterShell committed:
Moving: 1ic.BPMN-RPAstudio version 21.8.10 (#87583)

SpecterShell committed:
Moving: 1ic.BPMN-RPAstudio version 21.8.11 (#87584)

SpecterShell committed:
Moving: 1ic.BPMN-RPAstudio version 21.8.4 (#87585)

SpecterShell committed:
Moving: 1ic.BPMN-RPAstudio version 21.8.7 (#87587)

wingetbot committed:
Automatic deletion of Mega.MEGASync 4.7.1.0 (#87591)

* Updating manifests/m/Mega/MEGASync/4.7.1.0/Mega.MEGASync.installer.yaml

* Updating manifests/m/Mega/MEGASync/4.7.1.0/Mega.MEGASync.locale.en-US.yaml

* Updating manifests/m/Mega/MEGASync/4.7.1.0/Mega.MEGASync.yaml

SpecterShell committed:
ReleaseNotes: Principle.Lusun version 1.8.7 (#87592)

SpecterShell committed:
ReleaseNotes: FengHe.FocusNote version 2.16.1-678 (#87593)

SpecterShell committed:
Moving: 1ic.BPMN-RPAstudio version 21.8.6 (#87586)

oxygen-dioxide committed:
ovito.ovito-basic version 3.1.2 (#87595)

SpecterShell committed:
ReleaseDate: Wondershare.PDFelement.9 version 9.1.0 (#87597)

SpecterShell committed:
ReleaseNotes: Wondershare.PDFelement.9 version 9.2.0 (#87594)

Trenly committed:
Update VCRedist Dependency (#81836)

ImJoakim committed:
Remove: NordPassTeam.NordPass version 4.33.7 (#87808)

ImJoakim committed:
New version: NordPassTeam.NordPass version 4.33.9 (#87807)

jedieaston committed:
Updated Visual Studio Community 2022 to version 17.4.0. (#87817)

oxygen-dioxide committed:
ChemAxon.Marvin version 22.19.0 (#87602)

warshanks committed:
Update Ascension.AscensionLauncer to 1.95 (#87551)

Trenly committed:
Add AppsAndFeaturesEntries for VirusTotal Uploader (#87703)

* Add AppsAndFeaturesEntries for VirusTotal Uploader

* Fix PR

jedieaston committed:
Updated Visual Studio Test Agent 2022 to version 17.4.0. (#87821)

ImJoakim committed:
New version: Google.Drive version 66.0.3.0 (#87700)

jedieaston committed:
Updated Visual Studio Load Test Controller 2022 to version 17.4.0. (#87822)

jedieaston committed:
Added Visual Studio Enterprise 2022 version 17.4.0. (#87818)

Commit:
Remove manifest: RawTherapee.RawTherapee-Dev version 5.8-3162-g11168aa64 (#87809)

kylebavis committed:
Devolutions.RemoteDesktopManager version 2022.3.16.0 (#87696)

Commit:
New version: Overwolf.CurseForge version 0.212.1-6988 (#87690)

Commit:
New version: AOMEI.Backupper version 7.1.0 (#87722)

Commit:
New version: AnyLogic.AnyLogic.Professional version 8.8.1 (#87670)

infinitepower18 committed:
Google.PlayGames.Beta version 22.10.1285.8 (#87550)

* Google.PlayGames version 22.10.1285.8

* Update manifests/g/Google/PlayGames/22.10.1285.8/Google.PlayGames.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/g/Google/PlayGames/22.10.1285.8/Google.PlayGames.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/g/Google/PlayGames/22.10.1285.8/Google.PlayGames.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/g/Google/PlayGames/22.10.1285.8/Google.PlayGames.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/g/Google/PlayGames/22.10.1285.8/Google.PlayGames.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* fix

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

Commit:
New version: AnyLogic.AnyLogic.University version 8.8.1 (#87669)

jedieaston committed:
Added Visual Studio Build Tools 2022 version 17.4.0. (#87816)

Commit:
New version: AnyLogic.AnyLogic.Personal version 8.8.1 (#87668)

SpecterShell committed:
New version: Readdle.Spark version 3.1.0.38675 (#87796)

SpecterShell committed:
New version: Tauri.create-tauri-app version 2.6.2 (#87795)

SpecterShell committed:
Modify: Lapce.Lapce version 0.2.2 (#87769)

* Modify: Lapce.Lapce version 0.2.2

* Fix ReleaseNotesUrl

Commit:
New version: EclipseFoundation.TheiaBlueprint version 1.31.1 (#87667)

Commit:
New version: Google.AndroidStudio.Canary version 2022.2.1.8 (#87726)

SpecterShell committed:
New version: BartoszCichecki.LenovoLegionToolkit version 2.6.5 (#87797)

SpecterShell committed:
New version: Qingfeng.HeyboxWow version 1.9.7 (#87759)

SpecterShell committed:
New version: Catsxp.Catsxp version 107.2.11.4 (#87768)

SpecterShell committed:
New version: JinweiZhiguang.MasterAgent version 0.4.2.0 (#87764)

SpecterShell committed:
ReleaseNotes: Alibaba.Yuque version 1.8.4 (#87749)

SpecterShell committed:
New version: TailwindLabs.TailwindCSS version 3.2.2 (#87698)

* New version: TailwindLabs.TailwindCSS version 3.2.2

* Update ShortDescription

SpecterShell committed:
New version: Serverless.Serverless version 3.24.0 (#87664)

SpecterShell committed:
ReleaseNotes: Tencent.QQMusic version 18.97 (#87750)

SpecterShell committed:
New version: Serverless.Serverless version 3.24.1 (#87697)

SpecterShell committed:
New version: Apifox.Apifox version 2.2.7 (#87758)

SpecterShell committed:
Modify: Kingsoft.WPSOffice.CN version 11.1.0.12763 (#87622)

SpecterShell committed:
New version: IvanG.BrowserTamer version 2.8.0 (#87606)

SpecterShell committed:
New version: Ximalaya.XimalayaLive version 4.13.425 (#87607)

chrisbra committed:
New version: vim.vim version 9.0.0844 (#87728)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Commit:
New version: OpenWhisperSystems.Signal version 5.63.0 (#87666)

ImJoakim committed:
New version: Glarysoft.GlaryUtilities version 5.197.0.226 (#87701)

GeoffreyChen777 committed:
New version: GeoffreyChen.Paperlib version 2.1.1 (#87675)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

ilike2burnthing committed:
New version: Jackett.Jackett version 0.20.2210 (#87644)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Jaifroid committed:
WikiMed by Kiwix Electron v2.2.0-E (#87615)

elibroftw committed:
New version: ElijahLopez.MusicCaster version 5.9.1 (#87744)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Alphrag committed:
Ferdium.Ferdium.Nightly version 6.2.1-nightly.41 (#87678)

Jaifroid committed:
Kiwix.WikiMed version 2.2.0.0 (#87614)

nvim-winget committed:
Update: Neovim.Neovim.Nightly version 0.9.0 (#87609)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

JanDeDobbeleer committed:
JanDeDobbeleer.OhMyPosh version 12.13.0 (#87630)

sn4k3 committed:
PTRTECH.UVtools version 3.8.2 (#87634)

sprout2000 committed:
New version: sprout2000.LeafView version 2.2.12 (#87639)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

wingetbot committed:
Automatic update of AdGuard.AdGuardVPN from 2.0.690.0 to 2.0.706.0 (#87823)

* Creating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.installer.yaml

* Creating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.locale.en-US.yaml

* Creating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.yaml

* Updating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.installer.yaml

* Updating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.locale.en-US.yaml

* Updating manifests/a/AdGuard/AdGuardVPN/2.0.706.0/AdGuard.AdGuardVPN.yaml

matbech committed:
SmartSoft.SmartFTP version 10.0.3025.0 (#87608)

lepapareil committed:
Orange-OpenSource.Hurl version 1.8.0 (#87707)

lszomoru committed:
Microsoft.VisualStudioCode.Insiders_v1.74.0 (#87673)

Co-authored-by: Ladislau Szomoru <lszomoru@users.noreply.github.com>

john-preston committed:
New version: Telegram.TelegramDesktop version 4.3.0 (#87628)

Co-authored-by: telegramdesktop <6113871+telegramdesktop@users.noreply.github.com>

daiyam committed:
New version: VSCodium.VSCodium.Insiders version 1.73.0.22306 (#87620)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

Alphrag committed:
Ferdium.Ferdium.Nightly version 6.2.1-nightly.39 (#87642)

DavidXanatos committed:
New version: Sandboxie.Classic version 5.60.3 (#87747)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

chrisbra committed:
New version: vim.vim version 9.0.0841 (#87677)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

jedieaston committed:
Updated Visual Studio Team Explorer 2022 to version 17.4.0. (#87820)

SpecterShell committed:
New version: ByteDance.Feishu version 5.24.9 (#87752)

Commit:
New version: Google.AndroidStudio.Beta version 2022.1.1.15 (#87727)

wingetbot committed:
Automatic update of GeoGebra.CalculatorSuite from 6.0.741 to 6.0.742 (#87735)

* Creating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.installer.yaml

* Creating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.locale.en-US.yaml

* Creating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.yaml

* Updating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.installer.yaml

* Updating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.locale.en-US.yaml

* Updating manifests/g/GeoGebra/CalculatorSuite/6.0.742/GeoGebra.CalculatorSuite.yaml

chrisbra committed:
New version: vim.vim version 9.0.0836 (#87635)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

gtopala committed:
Remove: TopalaSoftwareSolutions.SIW version 2022.08.17 (#87662)

wingetbot committed:
Automatic update of GeoGebra.Classic from 6.0.741 to 6.0.742 (#87734)

* Creating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.installer.yaml

* Creating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.locale.en-US.yaml

* Creating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.yaml

* Updating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.installer.yaml

* Updating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.locale.en-US.yaml

* Updating manifests/g/GeoGebra/Classic/6.0.742/GeoGebra.Classic.yaml

Trenly committed:
Update VCRedist Dependency (#81837)

Commit:
Update LibreWolf.LibreWolf to v106.0.5-1 (#87626)

Co-authored-by: LibreWolf <publish_lw_win@librewolf.net>

jedieaston committed:
Added Visual Studio Professional 2022 version 17.4.0. (#87819)

wingetbot committed:
Automatic update of Google.ChromeRemoteDesktop from 107.0.5304.19 to 108.0.5359.16 (#87738)

* Creating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.installer.yaml

* Creating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.locale.en-US.yaml

* Creating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.installer.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.locale.en-US.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/108.0.5359.16/Google.ChromeRemoteDesktop.yaml

wingetbot committed:
Automatic update of GeoGebra.GraphingCalculator from 6.0.741 to 6.0.742 (#87736)

* Creating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.installer.yaml

* Creating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.locale.en-US.yaml

* Creating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.yaml

* Updating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.installer.yaml

* Updating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.locale.en-US.yaml

* Updating manifests/g/GeoGebra/GraphingCalculator/6.0.742/GeoGebra.GraphingCalculator.yaml

wingetbot committed:
Automatic update of BitSum.ProcessLasso.Beta from 12.0.0.19 to 12.0.0.21 (#87723)

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.installer.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.0.0.21/BitSum.ProcessLasso.Beta.yaml

oxygen-dioxide committed:
ovito.ovito-pro version 3.1.2 (#87598)

wingetbot committed:
Automatic update of Huawei.HuaweiCloudMeeting 8.10.5 (#87640)

* Updating manifests/h/Huawei/HuaweiCloudMeeting/8.10.5/Huawei.HuaweiCloudMeeting.installer.yaml

* Updating manifests/h/Huawei/HuaweiCloudMeeting/8.10.5/Huawei.HuaweiCloudMeeting.locale.en-US.yaml

* Updating manifests/h/Huawei/HuaweiCloudMeeting/8.10.5/Huawei.HuaweiCloudMeeting.locale.zh-CN.yaml

* Updating manifests/h/Huawei/HuaweiCloudMeeting/8.10.5/Huawei.HuaweiCloudMeeting.yaml

wingetbot committed:
Automatic update of Jidifudi.GT4T from 8.38.221101 to 8.38.221107 (#87679)

* Creating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.installer.yaml

* Creating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.locale.en-US.yaml

* Creating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.locale.zh-CN.yaml

* Creating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.yaml

* Updating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.installer.yaml

* Updating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.locale.en-US.yaml

* Updating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.locale.zh-CN.yaml

* Updating manifests/j/Jidifudi/GT4T/8.38.221107/Jidifudi.GT4T.yaml

wingetbot committed:
Automatic deletion of Qoppa.PDFStudio 2022.1.1.0 (#87646)

* Updating manifests/q/Qoppa/PDFStudio/2022.1.1.0/Qoppa.PDFStudio.installer.yaml

* Updating manifests/q/Qoppa/PDFStudio/2022.1.1.0/Qoppa.PDFStudio.locale.en-US.yaml

* Updating manifests/q/Qoppa/PDFStudio/2022.1.1.0/Qoppa.PDFStudio.yaml

wingetbot committed:
Automatic deletion of VirtualDesktop.Streamer 1.24.0 (#87617)

* Updating manifests/v/VirtualDesktop/Streamer/1.24.0/VirtualDesktop.Streamer.installer.yaml

* Updating manifests/v/VirtualDesktop/Streamer/1.24.0/VirtualDesktop.Streamer.locale.en-US.yaml

* Updating manifests/v/VirtualDesktop/Streamer/1.24.0/VirtualDesktop.Streamer.yaml

wingetbot committed:
Automatic update of SFLinux.Jami.Beta from 1.0.8335.25934 to 1.0.8346.37333 (#87765)

* Creating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.installer.yaml

* Creating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.locale.en-US.yaml

* Creating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.yaml

* Updating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.installer.yaml

* Updating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.locale.en-US.yaml

* Updating manifests/s/SFLinux/Jami/Beta/1.0.8346.37333/SFLinux.Jami.Beta.yaml

wingetbot committed:
Automatic update of ViewSonic.myViewBoard.Whiteboard from 2.60.3.0 to 2.61.3.0 (#87691)

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.installer.yaml

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.locale.en-US.yaml

* Creating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.installer.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.locale.en-US.yaml

* Updating manifests/v/ViewSonic/myViewBoard/Whiteboard/2.61.3.0/ViewSonic.myViewBoard.Whiteboard.yaml

wingetbot committed:
Automatic update of KCSoftwares.DUMo from 2.25.1.121 to 2.25.2.122 (#87680)

* Creating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.installer.yaml

* Creating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.locale.en-US.yaml

* Creating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.yaml

* Updating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.installer.yaml

* Updating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.locale.en-US.yaml

* Updating manifests/k/KCSoftwares/DUMo/2.25.2.122/KCSoftwares.DUMo.yaml

vedantmgoyal9 committed:
New version: Bilibili.Livehime version 4.32.0.4191 (#87366)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Automatic deletion of VirtualHere.USBServer 4.4.4 (#87618)

* Updating manifests/v/VirtualHere/USBServer/4.4.4/VirtualHere.USBServer.installer.yaml

* Updating manifests/v/VirtualHere/USBServer/4.4.4/VirtualHere.USBServer.locale.en-US.yaml

* Updating manifests/v/VirtualHere/USBServer/4.4.4/VirtualHere.USBServer.yaml

wingetbot committed:
Automatic update of GeoGebra.CASCalculator from 6.0.741 to 6.0.742 (#87733)

* Creating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.installer.yaml

* Creating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.locale.en-US.yaml

* Creating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.yaml

* Updating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.installer.yaml

* Updating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.locale.en-US.yaml

* Updating manifests/g/GeoGebra/CASCalculator/6.0.742/GeoGebra.CASCalculator.yaml

wingetbot committed:
Automatic deletion of WisdomSoftware.ScreenHunter 7.0.1435 (#87650)

* Updating manifests/w/WisdomSoftware/ScreenHunter/7.0.1435/WisdomSoftware.ScreenHunter.installer.yaml

* Updating manifests/w/WisdomSoftware/ScreenHunter/7.0.1435/WisdomSoftware.ScreenHunter.locale.en-US.yaml

* Updating manifests/w/WisdomSoftware/ScreenHunter/7.0.1435/WisdomSoftware.ScreenHunter.yaml

wingetbot committed:
Automatic update of RingCentral.RingCentral from 22.3.30.5037 to 22.3.35.5295 (#87685)

* Creating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.installer.yaml

* Creating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.locale.en-US.yaml

* Creating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.yaml

* Updating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.installer.yaml

* Updating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.locale.en-US.yaml

* Updating manifests/r/RingCentral/RingCentral/22.3.35.5295/RingCentral.RingCentral.yaml

vedantmgoyal9 committed:
New version: stevencohn.OneMore version 5.3.0 (#87815)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: alagrede.znote version 1.7.10 (#87798)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
Update: KDE.kdevelop version 5.6.2 (#87812)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

wingetbot committed:
Automatic deletion of GeoGebra.CASCalculator 6.0.741 (#87737)

* Updating manifests/g/GeoGebra/CASCalculator/6.0.741/GeoGebra.CASCalculator.installer.yaml

* Updating manifests/g/GeoGebra/CASCalculator/6.0.741/GeoGebra.CASCalculator.locale.en-US.yaml

* Updating manifests/g/GeoGebra/CASCalculator/6.0.741/GeoGebra.CASCalculator.yaml

wingetbot committed:
Automatic deletion of KCSoftwares.SUMo 5.16.2.527 (#87641)

* Updating manifests/k/KCSoftwares/SUMo/5.16.2.527/KCSoftwares.SUMo.installer.yaml

* Updating manifests/k/KCSoftwares/SUMo/5.16.2.527/KCSoftwares.SUMo.locale.en-US.yaml

* Updating manifests/k/KCSoftwares/SUMo/5.16.2.527/KCSoftwares.SUMo.yaml

fdncred committed:
Nushell.Nushell version 0.71.0 (#87826)

wingetbot committed:
Automatic deletion of Microsoft.WindowsVirtualDesktopAgent 1.0.4739.1000 (#87605)

* Updating manifests/m/Microsoft/WindowsVirtualDesktopAgent/1.0.4739.1000/Microsoft.WindowsVirtualDesktopAgent.installer.yaml

* Updating manifests/m/Microsoft/WindowsVirtualDesktopAgent/1.0.4739.1000/Microsoft.WindowsVirtualDesktopAgent.locale.en-US.yaml

* Updating manifests/m/Microsoft/WindowsVirtualDesktopAgent/1.0.4739.1000/Microsoft.WindowsVirtualDesktopAgent.yaml

elibroftw committed:
New version: ElijahLopez.MusicCaster version 5.9.0 (#87743)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

DavidXanatos committed:
New version: Sandboxie.Plus version 1.5.3 (#87748)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

john-preston committed:
New version: Telegram.TelegramDesktop version 4.3.1 (#87729)

Co-authored-by: telegramdesktop <6113871+telegramdesktop@users.noreply.github.com>

borque committed:
Mozilla.Thunderbird version 102.4.2 (#87619)

* Mozilla.Thunderbird version 102.4.2

* Update Mozilla.Thunderbird.locale.de-DE.yaml

Added Tags and Moniker

* Update Mozilla.Thunderbird.locale.en-US.yaml

Added Moniker

vedantmgoyal9 committed:
New version: JinweiZhiguang.MasterGo version 1.4.0 [FP-D] (#87423)

* New version: JinweiZhiguang.MasterGo version 1.4.0

* Update JinweiZhiguang.MasterGo.locale.en-US.yaml

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

borque committed:
Betterbird.Betterbird version 102.4.2-bb23 (#87627)

* Betterbird.Betterbird version 102.4.2-bb23

* Update Betterbird.Betterbird.locale.de-DE.yaml

Trenly committed:
Update VCRedist Dependency (#81839)

wingetbot committed:
Automatic deletion of Freelancer.Desktop 2.6.1 (#87637)

* Updating manifests/f/Freelancer/Desktop/2.6.1/Freelancer.Desktop.installer.yaml

* Updating manifests/f/Freelancer/Desktop/2.6.1/Freelancer.Desktop.locale.en-US.yaml

* Updating manifests/f/Freelancer/Desktop/2.6.1/Freelancer.Desktop.yaml

vedantmgoyal9 committed:
New version: foxglove.studio version 1.31.0 (#87828)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
Update: KDE.KDEConnect version 22.08.3 (#87830)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Element.Element version 1.11.14 (#87827)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

vedantmgoyal9 committed:
New version: Prey.Prey version 1.10.11 (#87814)

Co-authored-by: vedantmgoyal2009[bot] <110876359+vedantmgoyal2009[bot]@users.noreply.github.com>

borque committed:
geeksoftwareGmbH.PDF24Creator version 11.7.0 (#87631)

* geeksoftwareGmbH.PDF24Creator version 11.7.0

* Update geeksoftwareGmbH.PDF24Creator.locale.de-DE.yaml

Corrected URL

wingetbot committed:
Automatic update of PassMark.DiskCheckup from 3.5.1002.0 to 3.5.1003.0 (#87684)

* Creating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.installer.yaml

* Creating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.locale.en-US.yaml

* Creating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.locale.zh-CN.yaml

* Creating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.yaml

* Updating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.installer.yaml

* Updating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.locale.en-US.yaml

* Updating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.locale.zh-CN.yaml

* Updating manifests/p/PassMark/DiskCheckup/3.5.1003.0/PassMark.DiskCheckup.yaml

akohlmey committed:
Version 3Nov2022 of LAMMPS (#87657)

akohlmey committed:
Version 3Nov2022 of LAMMPS.LAMMPS-ML-PACE (#87659)

borque committed:
Mozilla.Firefox.ESR version 102.4.0 (#87633)

* Mozilla.Firefox.ESR version 102.4.0

* Update Mozilla.Firefox.ESR.locale.de-DE.yaml

Added additional URLs and information

* Update Mozilla.Firefox.ESR.locale.en-US.yaml

Added additional URLs and information

dorssel committed:
Update: dorssel.usbipd-win version 2.0.0 (#87654)

In preparation of switching to a different license in master:
- update LicenseUrl to point to the exact license for this version

Also backport the "no-auto-reboot" switches from version >= 2.3.0.

dorssel committed:
Update: dorssel.usbipd-win version 2.2.0 (#87652)

In preparation of switching to a different license in master:
- update LicenseUrl to point to the exact license for this version
Also backport the "no-auto-reboot" switches from version >= 2.3.0.

dorssel committed:
Update: dorssel.usbipd-win version 2.4.1 (#87649)

In preparation of switching to a different license in master:
- update LicenseUrl to point to the exact license for this version

xushengfeng committed:
Update: xushengfeng.eSearch version 1.9.1 (#87663)

* Update: xushengfeng.eSearch version 1.7.1

* Update: xushengfeng.eSearch version 1.7.2

* Update: xushengfeng.eSearch version 1.7.3

* Update: xushengfeng.eSearch version 1.7.4

* Update: xushengfeng.esearch version 1.8.0

* Update: xushengfeng.eSearch version 1.9.1

gerardog committed:
New version: gerardog.gsudo version 2.0.0 (#87676)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

akohlmey committed:
Version 3Nov2022 of LAMMPS.LAMMPS-USER-AEAM (#87658)

akohlmey committed:
Version 3Nov2022 of LAMMPS.LAMMPS-USER-VCSGC (#87661)

dorssel committed:
Update: dorssel.usbipd-win version 2.3.0 (#87651)

In preparation of switching to a different license in master:
- update LicenseUrl to point to the exact license for this version

slonopotamus committed:
Slonopotamus.Stevedore version 0.9.0 (#87655)

dorssel committed:
Update: dorssel.usbipd-win version 2.1.0 (#87653)

In preparation of switching to a different license in master:
- update LicenseUrl to point to the exact license for this version

Also backport the "no-auto-reboot" switches from version >= 2.3.0.
